src/main/java/frc/robot/arm/Arm.java

```diff
@@ -54,24 +56,13 @@ public boolean isAtHomePostion() {
     }
 
     public boolean isAtAmpPosition() {
-        return isOnTarget() && armTargetName().equals("Amp");
+        return isOnTarget();
```

Comment:
This functionality needs to continue to exist so that the slow shooting for amp scoring is triggered properly. 

You can do this by checking the target angle against the constant and isOnTarget